### ISSUE TITLE
[bug]create inline volume error

### ISSUE THREAD

**Issue by halfme** [alibaba/open-local#123]
### Ⅰ. Issue Description
create inline volume error

### Ⅱ. Describe what happened
```
  Normal   Scheduled    18s               default-scheduler  Successfully assigned default/file-server-6b9b66fb7c-bqbgl to block4
  Warning  FailedMount  1s (x6 over 18s)  kubelet            MountVolume.SetUp failed for volume "webroot" : rpc error: code = Internal desc = NodePublishVolume(mountLvmFS): mount lvm volume csi-993f2d33ec8c5892c833a038cfafa0e364df02be647edc83bc2f66d5435871ae with path /var/lib/kubelet/pods/f65659a8-7271-4600-b43b-39d692967276/volumes/kubernetes.io~csi/webroot/mount with error: rpc error: code = Internal desc = Failed to run cmd: /bin/nsenter --mount=/proc/1/ns/mnt --ipc=/proc/1/ns/ipc --net=/proc/1/ns/net --uts=/proc/1/ns/uts  lvcreate -n csi-993f2d33ec8c5892c833a038cfafa0e364df02be647edc83bc2f66d5435871ae -L 1024m open-local-pool-0, with out: WARNING: ext4 signature detected on /dev/open-local-pool-0/csi-993f2d33ec8c5892c833a038cfafa0e364df02be647edc83bc2f66d5435871ae at offset 1080. Wipe it? [y/n]: [n]
  Aborted wiping of ext4.
  1 existing signature left on the device.
  Failed to wipe signatures on logical volume open-local-pool-0/csi-993f2d33ec8c5892c833a038cfafa0e364df02be647edc83bc2f66d5435871ae.
  Aborting. Failed to wipe start of new LV.
, with error: exit status 5
```

### Ⅲ.  Anything else we need to know?
createLvm（inline volume）
```
cmd := fmt.Sprintf("%s lvcreate -n %s -L %d%s %s", localtype.NsenterCmd, volumeID, pvSize, unit, vgName)
```
CreateLV
```
args := []string{localtype.NsenterCmd, "lvcreate", "-n", name, "-L", fmt.Sprintf("%db", size), "-W", "y", "-y"}
```

**Comment by TheBeatles1994** [comment]
Yes, this is a bug, we must add "-y" option when creating lv.

**Comment by TheBeatles1994** [comment]
fixed by #124
